tomb-common/src/types/blockstore/car/carv1/mod.rs

```diff
@@ -49,10 +49,9 @@ impl CAR {
     }
 
     /// Write out a CARv1 object, assuming the Writer is already seeked to the first byte of the CARv1
```

Comment:
Comment says Carv1, but indices are related to CarV2. Why are we working index logic into CarV1 stuff?

Reply:
We are not presently reading in the Index as it is stored in `CARv2`. To be able to write and read blocks as expected within both our `CARv2`s (which do not process the index) and our `CARv1`s (which do not even store an index) we build a "CARv1 Index" artificially. Doing so requires that we seek through nearly the entire car file, jotting down the CIDs and offsets of those blocks into a HashMap. This is not performant at scale, which is why I am implementing the CARv2 index next thing, but also this code _is_ related to CARv1, it's not the same index you are imagining.

Reply:
makes sense to me